src/murmur/Server.cpp

```diff
@@ -598,7 +598,6 @@ void Server::setLiveConf(const QString &key, const QString &value) {
 			qsWelcomeText = text;
 			MumbleProto::ServerConfig mpsc;
 			mpsc.set_welcome_text(u8(qsWelcomeText));
```

Comment:
```suggestion
```

These can be removed as well. If we don't send the message, then there is no point in constructing it in the first place :)

Reply:
Should this deletion also include line 598? (`qsWelcomeText = text;`)

Reply:
Exactly :)